docs/guide/basics/release-cycle.md

```diff
@@ -12,9 +12,35 @@ So assuming next version is 1.x the two-month cycle will look as following:
 - v1.x-RC.y - unstable version with cutting-edge features ready to test and feedback
 - v1.x.y - stable version of the software **ready for production use**.
 
-# How new features are merged
+## How new features are merged
 
 During RC features Pull Request with new features after feedback and acceptance are normally merged to `develop` branch. 
 After entering the Stabilization Phase we are tagging current develop branch, creating a `RC-x` (where `x` is a number of current version) branch from it and working on stabilization there. 
 During the stabilization phase new features are merged to develop branch and will be merged on next `RC` phase.
 
+## Release cycle flow
+
+### 1. Development phase
+
+The first phase of cycle we're mostly focusing on features and improvements. Branches in that phase should be created from actual `develop`, also PRs should be pointed to this branch. Changes merged to develop are available to test on https://test.storefrontcloud.io
+
+![Development phase](./assets/release-cycle-1.png)
+
+### 2. Release Candidate phase
+
+At some point, when milestone for next minor versions is completed, we're creating new branch from `develop` called `RC-x.y` (example: `rc-1.9`).
```

Comment:
I'd consider following Git-flow naming, so:
- `release/` prefix
- name of the branch is used to create a tag after merging (git-flow tools are doing this automatically)

So it will be `release/v1.9.0-rc.1`

Reply:
If it's okay we should link to some git-flow description and update the "How Vue Storefront versions are released" part too.

Reply:
that's great, i'll change that

Reply:
actually - with rc?
So what with fixes?

Idea was to have one RC branch, and tag it as next RC versions. And now we'll have multiple rc branches per single release; i think this would be confusing. Could you elaborate more how is that better?

Reply:
I guess having too many branches is a bad idea. One rc which is equvallent of current develop is ok. This tagging is not giving us any advantage

Reply:
That might be true, will be easier to target changes.
So the final version will be:
`release/v1.9.0-rc` - just on for the whole stablization phase
`hotfix/v1.9.1` - new branch for each hotfix of stable release
am I correct?